# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/mergeant/libgda-include-dir.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libmergeant-orig/mg-query.c	2004-12-06 03:40:38.485761864 +0100
+++ ./libmergeant/mg-query.c	2004-12-06 03:41:48.337142840 +0100
@@ -44,7 +44,7 @@
 #include "mg-server-aggregate.h"
 #include "mg-data-handler.h"
 
-#include <libgda/sql/sql_parser.h>
+#include <sql/sql_parser.h>
 #include "mg-qf-func.h"
 
 /* 
